src/Components/QuoteSharePage/index.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+interface Props {
+  bgColor: string;
+}
+
+const FormatQuoteWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 30px;
+`;
+
+const QuotedText = styled.div<Props>`
+  background: ${(props) => props.bgColor};
+  border: 2px solid black;
+  margin-right: 10px;
+`;
+
+const QuoteFormatButtons = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const TextButton = styled.button`
+  margin-bottom: 5px;
+  border: 2px solid black;
+  cursor: pointer;
+`;
+const ColorButton = styled.button`
+  margin-bottom: 5px;
+  border: 2px solid black;
+  cursor: pointer;
+`;
+const ForwardButton = styled.button`
+  border: 2px solid black;
+  cursor: pointer;
+`;
+
+export default function QuoteSharePage() {
+  const bgColorList = [
```

Comment:
```suggestion
  const BG_COLOR_LIST = [
```
You can define this outside the component into constants file or top of the file